pkg/cloud-provider/cloudapi/azure/azure_resourcegraph.go

```diff
@@ -33,19 +34,23 @@ const (
 
 // resourceGraph returns resource-graph SDK apiClient.
 func (p *azureServiceSdkConfigProvider) resourceGraph() (azureResourceGraphWrapper, error) {
-	baseClient := resourcegraph.New()
-	baseClient.Authorizer = p.authorizer
+	baseClient, err := resourcegraph.NewClient(p.cred, nil)
+	if err != nil {
+		fmt.Println("failed to create arm resource graph client")
```

Comment:
fmt.Println?? We need logger and error to be propagated up. also add return and check other client init

Reply:
Here just returning error is sufficient, caller is propagating it upwards and finally CPA reconciler gets the error. Should this error also be update in CPA too?

Other clients are returning the error already.